examples-gallery/plot_parallel_park.py

```diff
@@ -1,3 +1,4 @@
+# %%
```

Comment:
Not sure why this is added as the first line, should be removed.

Reply:
> Not sure why this is added as the first line, should be removed.

I have to add this in VSC so I can run the simulation  from the begining.
I will remove before I push